### PR TITLE
Display/facet physical_media field

### PR DESCRIPTION
**Pull request by corylown**
- Display PHYSICAL_MEDIA field in brief and full record views.
- Add facet to filter by PHYSICAL_MEDIA_FACET (only displays after RESOURCE_TYPE_FACET has been selected).

### CODE REVIEW

File: lib/trln_argon/controller_override.rb [hunk 1]
```diff
@@ -90,10 +90,6 @@ module ControllerOverride
                                label: TrlnArgon::Fields::AVAILABLE_FACET.label,
                                limit: true,
                                collapse: false
-        config.add_facet_field TrlnArgon::Fields::INSTITUTION_FACET.to_s,
```
Comment (kazymovae) [inline lib/trln_argon/controller_override.rb]: Why did you remove this facet?

Comment (corylown) [reply lib/trln_argon/controller_override.rb]: It's not a facet we plan to use in production and it kept coming up in meetings that it was confusing so I removed it.
